CustomKeystore/src/main/java/com/amazonaws/services/kinesisanalytics/KDAFlinkStreamingJob.java

```diff
@@ -37,23 +49,11 @@
 				new KeyedSerializationSchemaWrapper(new SimpleStringSchema());
 
 		// Configure FlinkProducer for exactly-once semantics
-		FlinkKafkaProducer<String> sink = new FlinkKafkaProducer<>(
+		return new FlinkKafkaProducer<>(
```

Comment:
Can we make this reference the Custom Flink Kafka producer?

Reply:
Done